manifests/config.pp

```diff
@@ -12,8 +12,8 @@
         file {  $::grafana::cfg_location:
           ensure  => file,
           content => template('grafana/config.ini.erb'),
-          owner   => 'grafana',
-          group   => 'grafana',
+          owner   => $::grafana::grafana_system_user,
+          group   => $::grafana::grafana_system_group,
```

Comment:
Please don't add `::` after `$`. It was required for puppet 3. We slowly try to get rid of it.